src/test/java/tutorly/testutil/AttendanceRecordBuilder.java

```diff
@@ -0,0 +1,75 @@
+package tutorly.testutil;
+
+import tutorly.model.attendancerecord.AttendanceRecord;
+import tutorly.model.person.Person;
+
+/**
+ * A utility class to help with building AttendanceRecord objects.
+ */
+public class AttendanceRecordBuilder {
+
+    private int studentId;
+    private int sessionId;
+    private boolean isPresent;
+
+    /**
+     * Creates a {@code AttendanceRecordBuilder} with the default details.
+     */
+    public AttendanceRecordBuilder() {
+        studentId = 0;
+        sessionId = 0;
+        isPresent = false;
+    }
+
+    /**
+     * Initializes the AttendanceRecordBuilder with the data of {@code attendanceRecordToCopy}.
+     */
+    public AttendanceRecordBuilder(tutorly.model.attendancerecord.AttendanceRecord attendanceRecordToCopy) {
```

Comment:
Does this parameter type have to be this verbose? What about this?
```suggestion
    public AttendanceRecordBuilder(AttendanceRecord attendanceRecordToCopy) {
```

Reply:
Thanks for catching the issue!